assets/css/pages.css

```
@@ -32237,7 +32237,7 @@ tr.show-desktop {
   color: #222 !important;
   margin-bottom: 8px !important;
 }
-._4pyial {
+.form-badge {
```

```suggestion
.form__badge {
```
di nyo po naupdate sa css paupdate nalang